utils/mock-data-loader/banks/index.js

```diff
@@ -11,6 +11,20 @@ const BANKS = [
       teamName: 'Barclays Payment Reporting Team',
       email: 'payment-officer4@ukexportfinance.gov.uk',
     },
+    utilisationReportPeriodSchedule: [
+      { startMonth: 1, endMonth: 1 },
+      { startMonth: 2, endMonth: 2 },
+      { startMonth: 3, endMonth: 3 },
+      { startMonth: 4, endMonth: 4 },
+      { startMonth: 5, endMonth: 5 },
+      { startMonth: 6, endMonth: 6 },
+      { startMonth: 7, endMonth: 7 },
+      { startMonth: 8, endMonth: 8 },
+      { startMonth: 9, endMonth: 9 },
+      { startMonth: 10, endMonth: 10 },
+      { startMonth: 11, endMonth: 11 },
+      { startMonth: 12, endMonth: 12 },
+    ],
```

Comment:
Could save this as a `MONTHLY_REPORT_PERIOD_SCHEDULE` or similar to simplify adding this everywhere.